assets/panel/debugger.properties

```diff
@@ -386,8 +387,9 @@ editor.removeBreakpoint=Remove breakpoint
 editor.editBreakpoint=Edit breakpoint
 
 # LOCALIZATION NOTE (editor.addConditionalBreakpoint): Editor gutter context
-# menu item for adding a breakpoint condition on a line.
-editor.addConditionalBreakpoint=Add conditional breakpoint
+# menu item for adding/editing a breakpoint condition on a line.
+editor.addConditionalBreakpoint=Add/Edit conditional breakpoint
+editor.addConditionalBreakpoint.accesskey=C
```

Comment:
Please use 'c' (lowercase), it's less confusing than relying on the [fallback](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/accesskey) C -> c.